package.json

```diff
@@ -47,11 +47,17 @@
     "@uphold/github-changelog-generator": "^0.8.0",
     "eslint": "^5.13.0",
     "eslint-config-uphold": "^0.2.0",
+    "husky": "^1.3.1",
     "jest": "^19.0.2",
     "koa": "^2.0.1",
     "supertest": "^3.0.0"
   },
   "engines": {
     "node": ">=7.6"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn lint"
```

Comment:
Add `lint-staged`, like on other projects using `husky`?

Reply:
Is it worth it for a project so small to add that? I purposefully didn't add it, but if there is demand for it..

Reply:
For small projects maybe the basic `pre-commit` is enough, instead of having to use the bigger `husky`?

Reply:
Sure, I didn't compare package sizes *chuckle*. Just that to add `lint-staged` I would need an extra package, for a feature that is not very useful in this project.

Reply:
Done.